modules/040-node-manager/hooks/internal/instance_types/openstack_instance_types.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2022 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package instance_types
+
+// OpenstackInstanceTypes generated from https://docs.openstack.org/nova/rocky/admin/flavors.html Default Flavors
+var OpenstackInstanceTypes = map[string]*InstanceType{
```

Comment:
InstanceTypes maps are global variables that can be accidentally mutated in runtime. Can we avoid it by e.g., writing them as functions?